packages/manager/modules/pci/src/projects/project/additional-ips/order/translations/Messages_fr_FR.json

```
@@ -0,0 +1,47 @@
+{
+  "pci_additional_ips_failoverip_order": "Acheter des IPs Failover Cloud",
+  "pci_additional_ips_failoverip_order_generate": "Générer le bon de commande",
+  "pci_additional_ips_failoverip_order_cancel": "Annuler",
+  "pci_additional_ips_failoverip_order_description": "Les adresses IPs Failover Cloud sont déplaçables d'un serveur à l'autre au sein d'un même projet Cloud et d'un même continent.",
+  "pci_additional_ips_failoverip_order_quantity": "Nombre d'IP Failover",
+  "pci_additional_ips_failoverip_order_instance": "Instance à associer",
+  "pci_additional_ips_failoverip_order_region": "Pays de l'IP Failover",
+  "pci_additional_ip_create": "Ajouter une nouvelle IP additionnelle",
+  "pci_additional_ip_create_description": "Exposez votre application avec une IP agnostique",
+  "pci_additional_ip_create_step_select_ip": "Sélectionnez le type d'IP",
+  "pci_additional_ip_create_step_attach_instance": "Attachez une IP à une instance ou un réseau spécifique",
+  "pci_additional_ip_create_step_attach_instance_label": "Sélectionnez une instance",
+  "pci_additional_ip_create_action_label": "Générer le bon de commande",
+  "pci_additional_ip_create_create_instance": "<a href=\"{{ url }}\">Créer une instance</a>",
+  "pci_additional_ip_create_step_summary": "Résumé",
+  "pci_additional_ip_select_network_label": "Sélectionnez un réseau/IP",
+  "pci_additional_ip_select_network_placeholder": "Sélectionnez un réseau/IP",
+
+  "pci_additional_ip_failover_ip": "IP fail-over (OVHcloud)",
+  "pci_additional_ip_failover_ip_description": "solution optimisée pour les opérations de basculement inter-produits OVHcloud. Configuration manuelle nécessaire, prise en charge des cas d'utilisation du cloud hybride. Payé d'avance.",
```

So if you think we dont need the " : " in previous comment, you have to update this one :
```suggestion
  "pci_additional_ip_failover_ip_description": "Solution optimisée pour les opérations de basculement inter-produits OVHcloud. Configuration manuelle nécessaire, prise en charge des cas d'utilisation du cloud hybride. Payé d'avance.",
```

---

This is the validated content. I will check with the feature owner and make the change.